reference/res/res.send.md

```diff
@@ -1,58 +1,35 @@
 # res.send()
 
-Send a simple response.  `statusCode` defaults to 200 ("OK").
+This method us used to send a string as a response.
 
-This method is used in the underlying implementation of most of the other terminal response methods.
+It is used in the underlying implementation of most of the other terminal response methods.
 
 ### Usage
 ```usage
-return res.send([statusCode,] body);
+return res.send([string]);
 ```
 
-
 ### Details
-This method performs a myriad of useful tasks for simple non-streaming responses such as automatically assigning the Content-Length unless previously defined and providing automatic HEAD and HTTP cache freshness support.
-
-When a Buffer is given the Content-Type is set to "application/octet-stream" unless previously defined as shown below:
-
-```javascript
-res.set('Content-Type', 'text/html');
-res.send(new Buffer('some html'));
-```
-When a String is given the Content-Type is set to "text/html":
-
-```javascript
-res.send('some html');
-```
-When an Array or Object is given Express will respond with the JSON representation:
 
-```javascript
-res.send({ user: 'tobi' })
-res.send([1,2,3])
-```
-Finally when a Number is given without any of the previously mentioned bodies, then a response body string is assigned for you. For example 200 will respond will the text "OK", and 404 "Not Found" and so on.
+This method can be used to send a string or XML.
 
-```javascript
-res.send(200)
-res.send(404)
-res.send(500)
-```
+When a string is given the Content-Type is set to "text/html".
 
+If no argument is provided, a status code is returned.
 
 ### Example
 ```javascript
-res.send(new Buffer('whoop'));
-res.send({ some: 'json' });
-res.send('some html');
-res.send(404, 'Sorry, we cannot find that!');
-res.send(500, { error: 'something blew up' });
-res.send(200);
+res.send('some string');
+
+res.send(`<some> XML </some>`);
```

Comment:
### Can we break this into three examples:


For example, to allow users to export their own data in compliance with [2018's new GDPR regulations in Europe](https://www.eugdpr.org/), you might send back some dynamic CSV-formatted data like this:
```javascript
// Send back some dynamic CSV-formatted data.
return res.send(`
some,csv,like,this
or,,like,this
`);
```

Or to respond with XML, e.g. for a sitemap:
```javascript
// Send down some dynamic XML-formatted data.
return res.send(`<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
  <url>
    <loc>http://sailsjs.com</loc>
    <lastmod>2018-03-28T17:02:23.688Z</lastmod>
    <changefreq>monthly</changefreq>
  </url>
</urlset>
`);
```

You can also send arbitrary plain text, and use any status code you like:

```javascript
// You can use any status code you like.
// (Defaults to 200 unless you specify something else.)
return res.status(420).send('Hello world!');
```